src/workerd/api/pyodide/requirements.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include <capnp/compat/json.h>
+#include <capnp/message.h>
+#include <kj/common.h>
+#include <kj/map.h>
+
+namespace workerd::api::pyodide {
+
+capnp::json::Value::Reader getField(
+    capnp::List<::capnp::json::Value::Field, capnp::Kind::STRUCT>::Reader &object,
+    kj::StringPtr name);
+
+kj::String canonicalizePythonPackageName(kj::StringPtr name);
+
+typedef kj::HashMap<kj::String, kj::Vector<kj::String>>
+    DepMap;  // map from requirement to list of dependencies
```

Comment:
Nit: can we move the comment to above the typedef so that the type name doesn't get newlined?